SPECS/msft-golang/msft-golang.spec

```diff
@@ -1,3 +1,15 @@
+Summary:        Go
+Name:           msft-golang
+Version:        1.19.12
```

Comment:
Please run the buddy build before the merge to ensure the package builds in the pipeline as well.

Reply:
Sure, let me send the details offline.